etc/ci/buildbot_steps.yml

```diff
@@ -53,11 +53,11 @@ linux-rel-wpt:
     - ./mach build --release --with-debug-assertions -p servo
     - ./mach test-wpt-failure
     - ./mach test-wpt --release --processes 24 --total-chunks 2 --this-chunk 1 --log-raw test-wpt.log --log-errorsummary wpt-errorsummary.log --always-succeed
+    - ./mach test-wpt --release --headless --timeout-multiplier=4 --log-raw test-wdspec.log --log-errorsummary wpt-errorsummary.log webdriver
```

Comment:
Rather than sharing the same log, we should use separate logs and run a second filter-intermittents command. Also, we need to add --always-succeed to avoid failing immediately if any failures occur.